tensorflow/lite/kernels/detection_postprocess.cc

```diff
@@ -367,6 +367,13 @@ void DecreasingPartialArgSort(const float* values, int num_values,
       [&values](const int i, const int j) { return values[i] > values[j]; });
 }
 
+void DecreasingArgSort(const float* values, int num_values, int* indices) {
+  std::iota(indices, indices + num_values, 0);
+  std::stable_sort(
```

Comment:
Probably worth a brief comment here as to why we need/want stable_sort.

Reply:
Done